minecode_pipelines/pipes/__init__.py

```diff
@@ -7,12 +7,25 @@
 # See https://aboutcode.org for more information about nexB OSS projects.
 #
 
-import os
+from aboutcode.hashid import PURLS_FILENAME
+import textwrap
 import saneyaml
-
+from aboutcode import hashid
+import os
+from datetime import datetime
 from pathlib import Path
 
-from aboutcode.hashid import PURLS_FILENAME
+from git import Repo
+import json
+
+from minecode_pipelines.miners import write_data_to_file
+
+
+VERSION = os.environ.get("VERSION", "")
```

Comment:
You should import VERSION from the minecode_pipelines module

`from minecode_pipelines import VERSION`